src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs

```diff
@@ -460,21 +460,21 @@ public Task GetCertificates()
                             using (context.OpenReadTransaction())
                                 localCertKeys = ServerStore.Cluster.GetCertificateKeysFromLocalState(context).ToList();
 
-                            foreach (var localCertKey in localCertKeys)
+                            using (context.OpenReadTransaction())
```

Comment:
Can we shorten the transaction time by performing the deserialization during the transaction and only then closing it?

Reply:
Doesn't matter that much, the overall duration is roughly the same